app/controllers/admin/submissions_controller.rb

```diff
@@ -66,7 +66,8 @@ def create
       @submission =
         SubmissionService.create_submission(
           submission_params.merge(state: 'submitted'),
-          submission_params[:user_id]
+          submission_params[:user_id],
+          authorized_artsy_token?(session[:access_token])
```

Comment:
The ticket states that admins should be able to create submissions for non-target artists, does this only apply to admins? If so, is it easier to check against `AdminUser.exists? (gravity_user_id: user)`

Reply:
I think we are not interested in whether user is an admin or not, because sometimes a collector (user) asks an Artsy admin to create a submission for them. In such a case, our admin would put collector as user in the submission, and user wouldn't be an admin, but we still want to allow submission with non-target supply artist. 

Basically, what we want: 
- do not automatically reject, if submission created in Convection (even if submission's user is not an admin)
- automatically reject, if submission is created by user via web or mobile and it's a non-target supply artist

Reply:
`authorized_artsy_token?(session[:access_token])`

⬆️  this works in pratice, but what do you think? is there a better way of doing it?

Reply:
That is, we want to do this only for submissions who come from outside? Maybe then it would be better to add this logic to create_submission_resolver?

Reply:
lemme try!